python/runtime/tensorflow/predict.py

```diff
@@ -175,43 +175,15 @@ def add_to_example(example, x, i):
                 example.features.feature[feature_name].int64_list.value.extend(
                     list(values))
         else:
-            if "feature_columns" in feature_columns:
```

Comment:
Why we should remove this block?

Reply:
The removed block is to calculate `fc`. And `fc` is used to do special check when field's meta is int but should be convert to float. I think this convert is not necessary any more. The real reason we convert it is we did not pass the type info in go code, so tf will treat the data as float. And now we have fixed it. This may be relative to https://github.com/sql-machine-learning/sqlflow/pull/1964;

Reply:
Seems `feature_column_names_map` is used only at here, we can remove it later.